### DISCUSSION TITLE
Application is not available

### DISCUSSION THREAD

**Discussion by VishantBhat78** [okd-project/okd#1294] [General]
We deployed an application on OKD. All pods are deployed on okd but I think they may be crushing. Kindly have a look. Its a multi project OKD
![image](https://user-images.githubusercontent.com/93981857/182411138-e5e7a7f8-0eaf-46af-9342-55bc3ca3dec4.png)

**Comment by fortinj66** [comment]
go look at the pods in the console and check the events for each.  They may give you a clue to the issue

**Comment by fortinj66** [comment]
or `oc get events` from the same namespace/project
